website/docs/cdktf/concepts/resources.mdx

```diff
@@ -370,6 +370,78 @@ import imports.kubernetes.namespace.NamespaceMetadata;
 
 </CodeTabs>
 
+## Refactoring & Renaming Resources
+
+When working with your infrastructure definitions and the need arises to refactor or rename resources without destroying and recreating them, you can leverage the `moveTo` function. This section outlines how to perform resource moves, rename resources, and enable foreach on existing resources in your CDKTF project.
+
+### Performing Resource Moves
+
+The `moveTo` function is available on all resources and is used for relocating a resource to the location specified by the string target `moveTarget`. To set the `moveTarget` for a resource, utilize the `addMoveTarget` function that is present on the resource to move to.
+
+```ts
+new S3Bucket(this, "test-bucket-move-to", {
+  bucket: "move-bucket-name",
+}).addMoveTarget("move-s3");
+
+new S3Bucket(this, "test-bucket-move-from", {
+  bucket: "move-bucket-name",
+}).moveTo("move-s3");
+```
+
+After deployment, both the resource being moved and the move target on the destination resource can be removed.
+
+#### Move Targets
+
+A `moveTarget` is accessible anywhere within the context of the stack where it is created. Whether it's set at the root of the stack or within a nested construct, you can access it accordingly.
```

Comment:
```suggestion
A `moveTarget` is accessible anywhere within the context of the stack where it is created, including the root of the stack and within a nested construct. Moving resources to a different stack is not supported. 
```
Is that more concise? Per the style guide, temporal language, such as "in a future version . . .", "at this time", etc. should be omitted.

Reply:
Will note to not use such language in the future, [included](https://github.com/hashicorp/terraform-cdk/pull/3152/commits/70e4146b79b73f51e3b568df7fd5918366cb5b49) 👍